### PR TITLE
Batch support

### PR DESCRIPTION
**Pull request by vietj**
The new API does not implement batching. This is a test now failing.

### CODE REVIEW

File: src/test/java/io/vertx/jdbcclient/JDBCPoolTest.java [hunk 1]
```diff
@@ -282,7 +284,7 @@ public void testSelectBlob(TestContext should) {
       .onSuccess(rows -> {
         should.assertEquals(1, rows.size());
         for (Row row : rows) {
-          should.assertNull(row.getString(0));
+          //should.assertNull(row.getString(0));
```
Comment (pmlopes) [inline src/test/java/io/vertx/jdbcclient/JDBCPoolTest.java:287]: @vietj this throw `ClassCastException` now, weren't we saying that toString is always valid?

Comment (vietj) [reply src/test/java/io/vertx/jdbcclient/JDBCPoolTest.java:287]: now this has changed recently.

Comment (vietj) [reply src/test/java/io/vertx/jdbcclient/JDBCPoolTest.java:287]: now it should throw CCE. `null` shall be returned when it's actually null SQL value

Comment (vietj) [reply src/test/java/io/vertx/jdbcclient/JDBCPoolTest.java:287]: @pmlopes ping

**Pull request merged**